build/ci/phase-template.yml

```diff
@@ -1,13 +1,17 @@
 parameters:
   name: ''
   buildScript: ''
+  dockerImage: ''
   queue: {}
 
 phases:
   - phase: ${{ parameters.name }}
     variables:
       _buildScript: ${{ parameters.buildScript }}
       _phaseName: ${{ parameters.name }}
+      # if dockerImage is not equal to '' then run under docker container
+      ${{ if ne(parameters.dockerImage, '') }}:
+        _PREVIEW_VSTS_DOCKER_IMAGE: ${{ parameters.dockerImage }}
```

Comment:
@chcosta - do you have any idea how long we will need to continue using this "preview" feature in VSTS? And when a replacement will be possible?